### ISSUE TITLE
Versioned releases

### ISSUE THREAD

**Issue by BEFH** [AI-sandbox/gnomix#27]
Could you please start creating versioned releases for ease of ensuring reproducible results?

**Comment by AlexIoannidis** [comment]
Yes, will do. Nothing significant has been changed since initial release except visualization updates and (recently) readme updates.

**Comment by BEFH** [comment]
If you would like, I can also provide a docker container and github CI script that will publish to DockerHub whenever you make a release.

https://github.com/BEFH/docker_gnomix

I can open a pull request if you'd like.

**Comment by 0xaf1f** [comment]
any update about this issue?

**Comment by BEFH** [comment]
I have not heard anything on this.
